src/handler/BallotHandler.ts

```diff
@@ -32,7 +32,15 @@ export class BallotHandler {
 
   public getBallots = ballotService.getBallots;
 
-  public getRunningBallot = ballotService.getRunningBallot;
+  public getRunningBallot = async () => {
```

Comment:
For gods sake, please just use a normal function.

Reply:
For god sake fix ur own code
Resolved in  [#52 For gods sake, I am using a normal method now](https://github.com/LeoTuet/backend-we-want-to-sleep/pull/54/commits/ec17f7ac247fcc6782d574760d8876a902d0009a)